uploader/uploader.go

```diff
@@ -65,9 +65,16 @@ func NewUploaderWithBackend(backend Backend) *Uploader {
 	return &Uploader{backend}
 }
 
+type uploadFileInfo struct {
+	src  string
+	dest string
+}
+
 // Upload of all of files in a directory but not its subdirectories. Performs the uploads in parallel.
```

Comment:
This `uploadDir`, it looked like it was parallized already? 
Is this change just to cap the worker count of the uploaders?

Reply:
Correct. As I noted on the PR - this code is not used by the uploadLogFiles on container.Stop() - I've left these changes in to cap the concurrency when this code does get used. But for now watcher.go is where the uploads happen.